content/terraform/terraform.tf

```diff
@@ -178,3 +178,25 @@ module "exhibitions_listener" {
   priority = "111"
   path = "/exhibitions/*"
 }
+
+module "articles_listener" {
+  source = "../../shared-infra/terraform/service_alb_listener"
+  alb_listener_https_arn = "${local.alb_listener_https_arn}"
+  alb_listener_http_arn = "${local.alb_listener_http_arn}"
+  target_group_arn = "${module.content.target_group_arn}"
+  priority = "113"
```

Comment:
This is why we're missing a number:
https://github.com/wellcometrust/wellcomecollection.org/pull/3503/files#diff-f1b06eb5cc7af606f6d9cc85b872b912R187